apps/common/lib/common.ex

```diff
@@ -0,0 +1,10 @@
+defmodule Common do
+  @moduledoc """
+  Documentation for Common.
+  """
+
+  @typedoc """
+  A map representation of `Database.Schema.ModTag`.
+  """
+  @type tag :: String.t()
```

Comment:
is there a reason this doesn't have its own module? it'd be nice to be able to match on `%Tag{}` rather than it just being a string. the other advantage is we could move the tag mapping table out of the database cfg and into a real module

Reply:
sounds logical.
I thought one module for a type, that is nothing more than a string would be overkill.

Reply:
this is just for the same reason i listed under image.ex; similar things should be implemented in similar ways